pkgs/by-name/ni/nix-plugin-pijul/package.nix

```diff
@@ -14,7 +14,7 @@
   nixOverride ? null,
   nix-plugin-pijul,
 }:
-let nix = if nixOverride != null then nixOverride else nixVersions.nix_2_18;
+let nix = if nixOverride != null then nixOverride else nixVersions.stable;
```

Comment:
This was done because it might not (and this case I think will not?) compile with newer Nix versions. I just PRed an update that raises this to 2.24: #360005

Reply:
Merged.